docs/using-the-jdbc-driver/using-plugins/UsingTheIamAuthenticationPlugin.md

```diff
@@ -24,13 +24,17 @@ IAM database authentication use is limited to certain database engines. For more
+| `wrapperPlugins`  | String  |    No    | Set to `"iam"` to enable AWS IAM database authentication                                                                                                                                                                                                                                                               | `iam`                                               |
+| `iamDefaultPort`  | String  |    No    | This property will override the default port that is used to generate the IAM token. The default port is determined based on the underlying driver protocol. For now, there is support for `jdbc:postgresql:` and `jdbc:mysql:`. Target drivers with different protocols will require users to provide a default port. | `1234`                                              |
+| `iamHost`         | String  |    No    | This property will override the default hostname that is used to generate the IAM token. The default hostname is derived from the connection string. This parameter is useful when users are connecting with custom endpoints.                                                                                         | `database.cluster-hash.us-east-1.rds.amazonaws.com` |
+| `iamRegion`       | String  |    No    | This property will override the default region that is used to generate the IAM token. The default region is parsed from the connection string.                                                                                                                                                                        | `us-east-2`                                         |
+| `iamExpiration`   | Integer |    No    | This property will override the default expiration time that is assigned to the generated IAM token. The default expiration time is set to be 15 minutes.                                                                                                                                                              | `600`                                               |
+| `sslMode`         | String  |    No    | This property is required when using the IAM Authentication plugin with the MariaDB driver.                                                                                                                                                                                                                            | `verify-ca`                                         |
+| `serverSslCert`   | String  |    No    | This property is required when using the IAM Authentication plugin with the MariaDB driver. The value of this property should be the path to a SSL certificate. The certificates can be found [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL.html).                                    | `path/to/certificate.pem`                           |
```

Comment:
These are MariaDB specific parameters, I would avoid putting them in this table since they are not applicable to other drivers. I would just mention it in the sample code.